modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -695,6 +697,23 @@ else if (objectFieldDBTypes.contains(dbType) &&
 		}
 	}
 
+	private void _validateDefaultValue(String businessType, String defaultValue)
+		throws PortalException {
+
+		if (Validator.isNull(defaultValue)) {
+			return;
+		}
+
+		if (!Objects.equals(
+				ObjectFieldConstants.BUSINESS_TYPE_PICKLIST, businessType) &&
+			Validator.isNotNull(defaultValue)) {
```

Comment:
I will remove this `Validator.isNotNull(defaultValue)` condition because it's always true after line 703.